pyccel/stdlib/cwrapper/cwrapper.c

```diff
@@ -0,0 +1,508 @@
+#include "cwrapper.h"
+
+
+
```

Comment:
add licensing header and a small description of the file contents.

Reply:
Do you think we need the description in the `.c` file or just in the `.h` file? In c I am not sure that it is necessary for a `.c` file which does not contain a `main` to have a description. What do you think?

Reply:
yes, it's better to include it in `.h` files.

Reply:
Licensing fixed  : 682b9a3